content/docs/getting_started/cleanup.md

```diff
@@ -1,29 +1,34 @@
 ---
-title: "Cleanup"
+title: "Uninstall OSM"
 description: "Uninstall OSM and the bookstore applications"
 type: docs
 weight: 6
 ---
 
-# Cleanup
+# Uninstall OSM
 
-To cleanup all resources created for the demo, the OSM control plane, SMI resources, and the sample applications need to be deleted.
+The articles in the getting start section outline installing OSM and installing sample applications. To uninstall all of these resources, delete the sample application and related SMI resources and uninstall the OSM control plan and cluster-wide OSM resources.
+
+## Delete the sample applications
+
+To delete the sample applications and SMI resources, delete their namespaces. For example:
 
-To uninstall the sample applications and SMI resources, delete their namespaces with the following command:
 ```bash
 kubectl delete ns bookbuyer bookthief bookstore bookwarehouse
 ```
 
-### Uninstall OSM control plane
+## Uninstall OSM control plane
+
+To uninstall OSM control plane, use `osm unistall mesh`.
 
-To uninstall OSM control plane, run the following command.
 ```bash
 osm uninstall mesh
 ```
 
-### Uninstall OSM cluster wide resources
+## Uninstall OSM cluster-wide resources
+
+To uninstall OSM cluster-wide resources, use `osm uninstall cluster-wide-resources`.
 
-To uninstall OSM cluster wide resources, run the following command.
 ```bash
 osm uninstall cluster-wide-resources
```

Comment:
I think we can delete the bash command (L32-L34) since it's mentioned in-line.

Reply:
Thank you @shalier 

Keeping the code fence to allow for easy copy/paste